build/README.md

```diff
@@ -25,11 +25,9 @@
 * [Apache Thrift 0.9.3](http://archive.apache.org/dist/thrift/0.9.3/)
```

Comment:
currently it requires JDK 8, please modify too.

Reply:
fixed